### DISCUSSION TITLE
How to redirect to specific page inside axios interceptor?

### DISCUSSION THREAD

**Discussion by mohammad-zr** [vercel/next.js#45102] [Help]
I have a .ts file for configuring axios (baseAddress, setting headers & interceptors) and use it to call my apis. I want to redirect user to specific pages on each error code. for example navigate to login page after 401 error code
but I have not access to useRouter() or Router.
is there any solution to handle this issue?

**Comment by GabenGar** [comment]
In what context do you call this interceptor?
